### ISSUE TITLE
There is no way to access the "Theme configuration" page after hiding the "theme settings" button

### ISSUE THREAD

**Issue by pacquo** [authentic-theme/authentic-theme#1518]
Hello,

I've just installed Webmin 1.972 using the minimal version, and then Authentic Theme version 19.72 that is, currently, the last version of Authentic shown on webmin.com. 

After customizing the theme I ran into a kind of circular problem.

Among the options for the Navigation Menu under the Theme Configuration page there is one called "Show theme settings button": selecting "no" the button is, obviously, removed from the navigation bar.

The problem is that after this there is no other way to access the Theme Configuration page, that was previously shown as a tab under "Webmin Configuration/Webmin Themes".
In other words, this option leads to closing oneself out.

Am I wrong or there is a solution for this problem?

**Comment by iliarostovtsev** [comment]
Hi,

> After customizing the themeI ran into a kind of circular problem.

The configuration link, if hidden from configuration page can be found from navigation search:

![image](https://user-images.githubusercontent.com/4426533/110138880-7d68cb00-7de3-11eb-9ccf-3c00777be81f.png)


.. or from the dashboard:

![image](https://user-images.githubusercontent.com/4426533/110138783-675b0a80-7de3-11eb-8136-8516e8275a52.png)

**Comment by pacquo** [comment]
Now I see, thanks for clarifying :)
